### PR TITLE
MIG: base_suspend_security: Migration to 11.0

### PR DESCRIPTION
**Pull request by BT-fgarbely**
MIG: base_suspend_security: Migration to 11.0

### CODE REVIEW

File: base_suspend_security/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from . import ir_rule
+from . import ir_model_access
+from . import res_users
+from . import base
```
Comment (cuongnmtm) [inline base_suspend_security/models/__init__.py]: Please order these imports.
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports

File: base_suspend_security/models/ir_model_access.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from odoo import models, api, tools
```
Comment (cuongnmtm) [inline base_suspend_security/models/ir_model_access.py]: Please order this.
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports

File: base_suspend_security/models/ir_rule.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from odoo import models, api
```
Comment (cuongnmtm) [inline base_suspend_security/models/ir_rule.py]: Please order this.
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports

**Review by cuongnmtm — APPROVED** [review]
Minor fix.

**Review by hieulucky111 — APPROVED** [review]

**Comment by jcdrubay** [comment]
@OCA/core-maintainers : could you please help to review?

File: base_suspend_security/__manifest__.py [hunk 4]
```diff
@@ -19,8 +18,8 @@
 ##############################################################################
 {
     "name": "Suspend security",
-    "version": "10.0.1.0.0",
-    "author": "Therp BV, Odoo Community Association (OCA)",
+    "version": "11.0.1.0.0",
+    "author": "Therp BV, brain-tec AG, Odoo Community Association (OCA)",
```
Comment (hbrunn) [inline base_suspend_security/__manifest__.py]: you're a contributor, not an author

**Review by hbrunn — CHANGES_REQUESTED** [review]

**Review by hbrunn — APPROVED** [review]

**Pull request merged**